File: src/main/java/com/p2p/lending/dao/BorrowmoneyDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Borrowmoney;

public interface BorrowmoneyDao extends BaseDao<Object, Borrowmoney> {
	// ygx
	public List<Borrowmoney> pagingList(Map<String, Object> map);
	
	//周旗  2017年3月2日09:42:30
	public void toaddborr(Borrowmoney borrowmoney);
	public List<Borrowmoney> updhuankuan();
	public Borrowmoney borrowget(Integer ids);
	//hjy
	List<Borrowmoney> selecthjy(Map<String, Object> map);
	int hjycount();
}

File: src/main/java/com/p2p/lending/dao/BorrowmoneyDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Borrowmoney

interface BorrowmoneyDao : BaseDao<Any, Borrowmoney> {
    // ygx
    fun pagingList(map: Map<String, Any>): List<Borrowmoney>

    //周旗  2017年3月2日09:42:30
    fun toaddborr(borrowmoney: Borrowmoney)

    fun updhuankuan(): List<Borrowmoney>
    fun borrowget(ids: Int?): Borrowmoney
    //hjy
    fun selecthjy(map: Map<String, Any>): List<Borrowmoney>

    fun hjycount(): Int
}